src/lib/registry.js

```diff
@@ -87,3 +90,23 @@ export function registerWidgetValueSerializer(widgetName, serializer) {
 export function getWidgetValueSerializer(widgetName) {
   return registry.widgetValueSerializers[widgetName];
 };
+
+/**
+ * Backend API
+ */
+export function registerBackends(name, BackendClass) {
```

Comment:
I think this should be singular, since you are only registering one backend at a time (with each function call). For example, we have `registerWidget`.

Reply:
I thought so too 👍